src/lib/rest/StringFilter.cpp

```diff
@@ -48,6 +96,13 @@ StringFilterItem::~StringFilterItem()
 {
   stringList.clear();
   numberList.clear();
+
+  LM_W(("FGM this is %s, destroying", stringValue.c_str()));
```

Comment:
Ooooh. This time was my fault!! :D

@kzangeli could you manage to grep for "FGM" and remove potential traces, pls? (you are now in the branch and it is easier for you).